### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by peterstratton** [weii41392/AQT#9]
Hello, 

I'm very interested in this work and have been attempting to reproduce the results, but even after fixing label mismatch issue in #4 , I have been unable to reproduce a mAP score of 47.1 after 50 epochs of training. Specifically, if I change r50_uda_c2fc.yaml to use batch size of 8, as described in the paper, I get a mAP score of 45.0. I trained using 1 GPU. However, I do not have the exact same system setup as described in the README. I'm using newer versions of Pytorch, Torchvision, and CUDA because my graphics card is too new and is not compatible with CUDA 9.2 anymore. 

If someone could point me in the right direction to reproduce the results of this paper, that would be great!

**Comment by suojinhui** [comment]
May I ask if you have used BoxRefine? The performance reported by the author largely comes from BoxRefine (a technology inherent in deformable DETR).

**Comment by yuguoliua** [comment]
hello，Have you tried running this code with multiple GPUs? I've tried several methods to train with multiple GPUs, but none of them have worked. However, I find that training with just one GPU is really too slow.I hope you can help me with this, and I would really appreciate it.

**Comment by yuguoliua** [comment]
> hello，Have you tried running this code with multiple GPUs? I've tried several methods to train with multiple GPUs, but none of them have worked. However, I find that training with just one GPU is really too slow.I hope you can help me with this, and I would really appreciate it.

hello，Have you tried running this code with multiple GPUs? I've tried several methods to train with multiple GPUs, but none of them have worked. However, I find that training with just one GPU is really too slow.I hope you can help me with this, and I would really appreciate it.

**Comment by yuguoliua** [comment]
> Hello,
> 
> I'm very interested in this work and have been attempting to reproduce the results, but even after fixing label mismatch issue in #4 , I have been unable to reproduce a mAP score of 47.1 after 50 epochs of training. Specifically, if I change r50_uda_c2fc.yaml to use batch size of 8, as described in the paper, I get a mAP score of 45.0. I trained using 1 GPU. However, I do not have the exact same system setup as described in the README. I'm using newer versions of Pytorch, Torchvision, and CUDA because my graphics card is too new and is not compatible with CUDA 9.2 anymore.
> 
> If someone could point me in the right direction to reproduce the results of this paper, that would be great!

Hi there, I've been trying to replicate this paper recently, but I've been unsuccessful despite many attempts. Could you share the files from your replication? I would really appreciate it!

**Comment by yuguoliua** [comment]
> Hello,
> 
> I'm very interested in this work and have been attempting to reproduce the results, but even after fixing label mismatch issue in #4 , I have been unable to reproduce a mAP score of 47.1 after 50 epochs of training. Specifically, if I change r50_uda_c2fc.yaml to use batch size of 8, as described in the paper, I get a mAP score of 45.0. I trained using 1 GPU. However, I do not have the exact same system setup as described in the README. I'm using newer versions of Pytorch, Torchvision, and CUDA because my graphics card is too new and is not compatible with CUDA 9.2 anymore.
> 
> If someone could point me in the right direction to reproduce the results of this paper, that would be great!
My average precision in the replication is 23, and I suspect this might be due to not loading the weights pretrained on ImageNet. Do I need to load the pretrained weights? I look forward to your response

**Comment by yuguoliua** [comment]
> 你好
> 
> 我对这项工作非常感兴趣，并一直在尝试重现结果，但即使在修复 #4 中的标签不匹配问题后，经过 50 个训练时期，我仍然无法重现 47.1 的 mAP 分数。具体来说，如果我将 r50_uda_c2fc.yaml 更改为使用批量大小 8，如论文中所述，我的 mAP 分数为 45.0。我使用 1 个 GPU 进行训练。但是，我没有与 README 中描述的完全相同的系统设置。我正在使用较新版本的 Pytorch、Torchvision 和 CUDA，因为我的显卡太新了，不再与 CUDA 9.2 兼容。
> 
> 如果有人能为我指出正确的方向来重现这篇论文的结果，那就太好了！

My average precision in the replication is 23, and I suspect this might be due to not loading the weights pretrained on ImageNet. Do I need to load the pretrained weights? I look forward to your response
